tedana/decomposition/pca.py

```diff
@@ -31,34 +30,6 @@
     """),
```

Comment:
The duecredit decorators associated with the MLE function need to removed as well.

Reply:
Yeah, I would have noticed if I'd run flake8 while following *my own checklist* 🤦‍♂. That's in an unpushed commit, waiting to make sure the integration tests to pass on Circle because my conda env is a little screwy (my fault).